sweep/store.go

```diff
@@ -186,49 +161,12 @@ func (s *sweeperStore) NotifyPublishTx(sweepTx *wire.MsgTx) error {
 			return err
 		}
 
-		if err := lastTxBucket.Put(lastTxKey, b.Bytes()); err != nil {
```

Comment:
If we want to just get the hash, we can just do `sweepTx.TxHash()` now to get that value.